examples/sampleCalls.cpp

```diff
@@ -1252,7 +1391,7 @@ TEST_SUITE("ZendooBatchProofVerifier") {
                 false, constant, sc_id, epoch_number, quality, NULL, 0,
                 NULL, 0, end_cum_comm_tree_root, btr_fee, ft_min_amount, sc_pk,
                 (path_char_t*)proof_path.c_str(), proof_path.size(), NUM_CONSTRAINTS,
-                &ret_code, true, &SUPPORTED_SEGMENT_SIZE
+                NULL, &ret_code, true, &SUPPORTED_SEGMENT_SIZE
```

Comment:
It would be nice to add test cases in which prev_cert_hash is present also in the ZendooBatchVerifier

Reply:
Done with e9273b2fefa804357570c0b4209949d6a81e9d26